test/lib/log.spec.js

```diff
@@ -0,0 +1,52 @@
+"use strict";
+
+/* eslint-disable no-console */
+
+const { setLoggingOptions, error, log, warn } = require("../../lib/log");
+const chai = require("chai");
+const sinon = require("sinon");
+const sinonChai = require("sinon-chai");
+chai.use(sinonChai);
+
+describe("lib/log", () => {
+  beforeEach(() => {
+    sinon.spy(console, "log");
```

Comment:
GLOBAL: Always try to avoid using `sinon.spy|stub` direct methods and instead use a single `sandbox` object that unwinds in a `beforeEach` like here: https://github.com/FormidableLabs/trace-pkg/blob/main/test/bin/lib/args.spec.js#L9-L19

Reply:
This has been converted to use the strategy outlined in other tests (sandboxes).  I tried again to add in a debuglog Spy / mock and it would not detect being called for some reason. Very weird.